Tests/OrderedCollectionsTests/OrderedDictionary/OrderedDictionary+Values Tests.swift

```diff
@@ -15,7 +15,40 @@ import XCTest
 import _CollectionsTestSupport
 
 class OrderedDictionaryValueTests: CollectionTestCase {
-  func test_values_getter() {
+  func test_values_getter_equal() {
+    let left: OrderedDictionary = [
+      "one": 1,
+      "two": 2,
+      "three": 3,
+      "four": 4,
+    ]
+    let right: OrderedDictionary = [
+      "one": 1,
+      "two": 2,
+      "three": 3,
+      "four": 4,
+    ]
+    expectEqual(left.values, right.values)
```

Comment:
We should also make sure to exercise the new code path:

```suggestion
    expectEqual(left.values, left.values) // Identity fast path
    expectEqual(left.values, right.values) // Linear algorithm
```